bumble/rfcomm.py

```diff
@@ -592,7 +621,13 @@ class Multiplexer(EventEmitter):
         RESET: 'RESET',
     }
 
-    def __init__(self, l2cap_channel, role):
+    connection_result: Optional[asyncio.Future]
+    disconnection_result: Optional[asyncio.Future]
+    open_result: Optional[asyncio.Future]
+    acceptor: Optional[Callable[[int], bool]]
+    dlcs: Dict[int, DLC]
+
+    def __init__(self, l2cap_channel, role) -> None:
```

Comment:
maybe `l2cap_channel` and `role` should be typed?

Reply:
Done. BTW, we have many enum-like integer fields which cannot be handled by typing.Literal.

Reply:
Let's resolve this issue later. These are all internal states, so I think we can use some enum to replace them.